Orlando Walking Tours/Controllers/LocationListVC.swift

```diff
@@ -133,7 +132,7 @@ class LocationListVC: UIViewController
             let locationIndex = self.locations.indexOf { $0.locationId == locationId }!
             print("add to tour \(locationIndex)")
             let location = self.locations.removeAtIndex(locationIndex)
-            modelService.addLocation(location, toTour: self.tour)
+            modelService.addLocation(location, toTour: self.tour!)
```

Comment:
self.tour! could cause a crash if by some chance tour gets dereferenced. It would be best to safely unwrap and and throw an error if it doesn't unwrap than crash the app.